CLA.md

```diff
@@ -13,19 +13,22 @@ It is important to read and understand this legal agreement.
 
 ## How do I sign?
 
-After creating your first Pull Request the linux-foundation-easycla bot will respond with information regarding your CLA status along with a link to sign the CLA.
+After creating your first Pull Request, the linux-foundation-easycla bot will respond with information regarding your CLA status along with a link to sign the CLA.
 
 <img width="1065" alt="EasyCLA bot" src="https://user-images.githubusercontent.com/69111235/152226443-f6fe61ee-0e92-46c5-b6ea-c0deb718a585.png">
 
-#### 1. If you are signing up as a corporate contributor, ensure that you have linked your corporate email address to your GitHub profile (it doesn't have to be your primary email address for GitHub) or else it can lead to issues with the CLA system. For more information, see [Adding an email address to your GitHub account](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-email-preferences/adding-an-email-address-to-your-github-account).
+### 1. If you are signing up as a corporate contributor, ensure that you have linked your corporate email address to your GitHub profile (it doesn't have to be your primary email address for GitHub) or else it can lead to issues with the CLA system.
```

Comment:
not sure, but maybe you can keep the `####` instead of `###`

Reply:
By keeping #### the difficulty I faced as a new contributor is that I was unable to differentiate between the heading and the paragraphs and it looked visually unpleasant and slightly messy.

Reply:
They are not really headings, but just steps in an instruction. I'd leave them as is as well.
The only reason they're a 'heading' instead of Bold text is to make it linkable.

Reply:
Thanks for your feedback! I'm leaving them as is then.